src/components/admin/Settings/StorageMappings/Dialog/Form.tsx

```diff
@@ -0,0 +1,34 @@
+import { JsonForms } from '@jsonforms/react';
+import { Stack, StyledEngineProvider } from '@mui/material';
+import useConfigurationSchema from 'components/admin/Settings/StorageMappings/Dialog/useConfigurationSchema';
+import { useStorageMappingStore } from 'components/admin/Settings/StorageMappings/Store/create';
+import { jsonFormsPadding } from 'context/Theme';
+import { isEmpty } from 'lodash';
+import { jsonFormsDefaults } from 'services/jsonforms/defaults';
+
+function StorageMappingForm() {
+    const formValue = useStorageMappingStore((state) => state.formValue);
+    const updateFormValue = useStorageMappingStore(
+        (state) => state.updateFormValue
+    );
+
+    const { schema, uischema } = useConfigurationSchema();
+
+    return Boolean(!isEmpty(schema) && !isEmpty(uischema)) ? (
```

Comment:
I know we're being safe - but I think we can remove this. Then in the `useConfigurationSchema` could probably work like a context hook where it just throws an error if `undefined` was going to be returned.